wiki/content/deploy/tls-configuration.md

```diff
@@ -192,7 +266,7 @@ The server will always **request** Client Authentication.  There are four differ
 | `VERIFYIFGIVEN`    | optional        | Client certificate is VERIFIED if provided (default) |
 | `REQUIREANDVERIFY` | required        | Client certificate is always VERIFIED (most secure) |
 
-{{% notice "note" %}} `REQUIREANDVERIFY` is the most secure but also the most difficult to configure for remote clients. When using this value, the value of `--tls_server_name` is matched against the certificate SANs values and the connection host.{{% /notice %}}
+{{% notice "note" %}} `REQUIREANDVERIFY` is the most secure but also the most difficult to configure for remote clients. This option is required if you encrypt both internal and external ports. When using this value, the value of `--tls_server_name` is matched against the certificate SANs values and the connection host.{{% /notice %}}
```

Comment:
I guess there is a confusion here, so internal ports if enabled via --tls_internal_port_enabled, it will always be REQUIREANDVERIFY. But for internal ports only. This means by default, internal ports 5080 and 7080 connect with mtls but 9080,8080,6080 uses VERIFYIFGIVEN as the setting. If while starting server `--tls_client_auth` option is provided it will only affect the external ports i.e. 9080, 8080, 6080. We need to clarify this